app/models/organization.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Organization < ApplicationRecord
+  MAX_NAME_LENGTH = 100
+
+  has_many :posts, dependent: :destroy
+
+  has_many :users, dependent: :destroy
+
+  has_many :categories, dependent: :destroy
+
+  validates :organization_name, presence: true, uniqueness: true, length: { maximum: MAX_NAME_LENGTH }
```

Comment:
```suggestion
  validates :organization_name, presence: true, uniqueness: true, length: { maximum: MAX_NAME_LENGTH }
```
Why do you have `organization_name` as your column name? It is understood right if the table organization has a name column, it would be the organization's name? Please name your columns sensibly.
Everytime you retrieve the name, it would be `organization.organization_name` which is not clean code.

Reply:
@jaimysimon I have fixed this issue for organizations. I also observed same issue with categories and fixed it.